blocks/library/gallery/index.js

```diff
@@ -118,9 +175,8 @@ registerBlockType( 'core/gallery', {
 					<MediaUploadButton
 						buttonProps={ uploadButtonProps }
 						onSelect={ setMediaUrl }
-						type="image"
-						multiple="true"
-						gallery="true"
+						frame={ GalleryMediaFrame }
```

Comment:
Instead of passing the frame object as a prop, should we create more "targeted" props? (like we do for the "multiple" or "value" (for the selected Ids)....? This would ease the usage of the component IMO.